E2ETests/README.md

```diff
@@ -0,0 +1,88 @@
+# End to End Tests
+
+E2E Test scenarios are run in [Synthetics for Mobile](https://docs.datadoghq.com/mobile_app_testing/) and [Monitors](https://docs.datadoghq.com/monitors/) will assert the proper propagation of data.
+
+
+## CI
+
+CI will continuously build, sign, and upload a Runner application to Synthetics which will run predefined Tests daily.
+
+### Build
+
+Before building the application, the `E2ETests/xcconfigs/E2E.local.xcconfig` configuration file must be present with the `Mobile - Integration Org` client token and RUM application ID, these values are sensitive and must be securely stored.
+
+```ini
+CLIENT_TOKEN=
+RUM_APPLICATION_ID=
+```
+
+> [!TIP]
+> Files can be base64 encoded and stored in secret variables. To copy the encoded file content, you can do:
+> ```bash
+> cat E2ETests/xcconfigs/E2E.local.xcconfig | base64 | pbcopy
+> ```
+
+### Sign
+
+To sign the Runner application, the Certificate and Provision Profile defined in [Synthetics.xcconfig](xcconfigs/Synthetics.xcconfig) and in [exportOptions.plist](exportOptions.plist) needs to be installed on the build machine, these files are sensitive and must be securely stored. Make sure to update both files when updating the Certificate and Provisioning Profile, otherwise signing will fail.
```

Comment:
```suggestion
To sign the runner application, the certificate and provision profile defined in [Synthetics.xcconfig](xcconfigs/Synthetics.xcconfig) and in [exportOptions.plist](exportOptions.plist) needs to be installed on the build machine. These files are sensitive and must be securely stored. Make sure to update both files when updating the certificate and provisioning profile, otherwise signing fails.
```